win32ss/user/user32/windows/defwnd.c

```diff
@@ -1036,12 +1036,14 @@ RealDefWindowProcW(HWND hWnd,
             if ( Wnd &&
                  Wnd->style & (WS_HSCROLL | WS_VSCROLL) )
             {
-               if (!Wnd->pSBInfo)
-               {
-                  SCROLLINFO si = {sizeof si, SIF_ALL, 0, 100, 0, 0, 0};
-                  SetScrollInfo( hWnd, SB_HORZ, &si, FALSE );
-                  SetScrollInfo( hWnd, SB_VERT, &si, FALSE );
-               }
+                if (!Wnd->pSBInfo)
+                {
+                    SCROLLINFO si = {sizeof si, SIF_ALL, 0, 100, 0, 0, 0};
+                    if (Wnd->style & WS_HSCROLL)
+                        SetScrollInfo( hWnd, SB_HORZ, &si, FALSE );
+                    if (Wnd->style & WS_VSCROLL)
+                        SetScrollInfo( hWnd, SB_VERT, &si, FALSE );
+                }
```

Comment:
Please follow existing indentation style to make diff clearer.

Reply:
@binarymaster  Done